detections/endpoint/suspiciously_named_executable.yml

```diff
@@ -0,0 +1,74 @@
+name: windows_randomized_filenames
```

Comment:
search name should be : Windows Randomized Filenames

change the file name to `windows_randomized_filenames.yml`. Same with the test file names and words.

Reply:
Done